bindings/py/cpp_src/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ message(STATUS "Configuring Python interface")
 #
 set(src_py_algorithms_files
     bindings/algorithms/algorithm_module.cpp
+    bindings/algorithms/py_SDR.cpp
```

Comment:
Very good 👍

Reply:
MSVC branch merged, this should be fixed.